docs/software/dns_servers.md

```diff
@@ -2,14 +2,114 @@
+- Also Installs: [Webserver stack](../webserver_stack/)
+
+![DietPi DNS server software Pi-hole](../assets/images/dietpi-software-dnsserver-pihole.png){: style="width:500px"}
+
+### Access the web interface
+
+The web interface of Pi-hole can be accessed via:
+
+- URL= http://<your.IP/domain/admin\>
+- Password = <yourGlobalSoftwarePassword\> (default: dietpi)
+
+### Configure devices to use Pi-hole for DNS resolution
+
+#### Option 1 - Setup single devices to use the Pi-hole DNS server
+
+Simply change your DNS settings to use the IP address of your Pi-hole device. This will need to be done for each device that you want Pi-hole to work with.
+
+Example:
+
+- My Pi-hole device has the IP address of 192.168.0.100
+- On my PC, I would set the DNS address to 192.168.0.100
+- Tutorial [The Ultimate Guide to Changing Your DNS settings](https://www.howtogeek.com/167533/the-ultimate-guide-to-changing-your-dns-server/).
+
+#### Option 2 - Setup your router to use the Pi-hole DNS server
+
+This method will automatically point every device (that uses DHCP) on your network to Pi-hole.
+On your routers control panel web page, you will need to find a option called "DNS server". This should be located under DHCP settings.
+
+Simply enter the IP address of your Pi-hole device under "DNS server":
+
+![DietPi DNS server software router setup](../assets/images/dietpi-software-dnsserver-router-setup.png){: style="width:500px"}
+
+On your Pi-hole device, you will need to set a different DNS server.  
+Depending on your router configuration, if you don't do this step, the Pi-hole device may not be able to access the internet. It's highly recommended to have the device running Pi-hole, pointing to a DNS server outside your network.
+
+- Run the following command: `dietpi-config 8 1`
+- Select: *Ethernet*
+- If you are running in DHCP mode, select *Change Mode*, then select: *Copy Current address to Static*
+- Select *Static DNS* from the list, then choose a DNS server, or manually enter a custom entry.
+- Once completed, select *Apply* to save the changes.
+
+### Further procedures
+
+#### Updating Pi-hole
+
+Pi-hole can be updated via the shell command `pihole -up`.
+
+#### Repairing Pi-hole
+
+You can use `pihole -r` to repair or reconfigure your Pi-hole instance.
+
+!!! warning "No selection of Lighttpd during repair procedure"
+    Do NOT select to install Lighttpd when being asked, as this will mix our own webserver stack setup with a different once provided by the Pi-hole installer, which causes various issues.
+
+#### Setting the password for Pi-hole
+
+If you forgot your login password for the Pi-hole admin web page, you can set it with the shell command `pihole -a -p` on your Pi-hole device.
+
+#### Accessing Pi-hole via OpenVPN or WireGuard
+
+To allow (OpenVPN or WireGuard) VPN clients accessing your local Pi-hole instance, you need to allow DNS requests from all network interfaces: `pihole -a -i local`.
+
+See also <https://de.wikipedia.org/wiki/Pi-hole>.
```

Comment:
Whoopsie, it shouldn't be the German wikipedia page, lets keep it international: https://wikipedia.org/wiki/Pi-hole
My fault when copy&paste the auto-localised web page.

Reply:
Changed.